.github/workflows/build-ios-release-pullrequest.yml

```
@@ -86,50 +93,67 @@ jobs:
         id: generate_build_number
         run: |
           NEW_BUILD_NUMBER=$(date +%s)
           echo "NEW_BUILD_NUMBER=$NEW_BUILD_NUMBER" >> $GITHUB_ENV
           echo "::set-output name=build_number::$NEW_BUILD_NUMBER"
+
       - name: Set Build Number
         run: bundle exec fastlane ios increment_build_number_lane
+
       - name: Determine Marketing Version
         id: determine_marketing_version
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:46: Double quote to prevent globbing and word splitting

(shellcheck)